src/test/java/uk/nhs/prm/deduction/e2e/models/ContinueRequestMessage.java

```diff
@@ -0,0 +1,43 @@
+package uk.nhs.prm.deduction.e2e.models;
+
+import uk.nhs.prm.deduction.e2e.utility.Resources;
+
+import java.util.UUID;
+
+import static uk.nhs.prm.deduction.e2e.utility.TestUtils.getUUIDAsUpperCaseString;
+
+public class ContinueRequestMessage {
+    private UUID conversationId;
+    private UUID messageId;
+    private String sourceGpOds;
+    private String destinationGpOds;
+    private String sourceGpAsid;
+    private String destinationGpAsid;
+
+    public ContinueRequestMessage(UUID conversationId, UUID messageId, String sourceGpOds, String destinationGpOds, String sourceGpAsid, String destinationGpAsid) {
+        this.conversationId = conversationId;
+        this.messageId = messageId;
+        this.sourceGpOds = sourceGpOds;
+        this.destinationGpOds = destinationGpOds;
+        this.sourceGpAsid = sourceGpAsid;
+        this.destinationGpAsid = destinationGpAsid;
+    }
+
+    public String toJsonString() {
+        return Resources.readTestResourceFile("COPC_IN000001UK01")
+                .replaceAll("DBC31D30-F984-11ED-A4C4-956AA80C6B4E", conversationId())
+                .replaceAll("DE304CA0-F984-11ED-808B-AC162D1F16F0", messageId())
+                .replaceAll("B85002", sourceGpOds)
+                .replaceAll("200000001613", sourceGpAsid)
+                .replaceAll("M85019", destinationGpOds)
+                .replaceAll("200000000149", destinationGpAsid);
+    }
+
+    public String conversationId() {
+        return getUUIDAsUpperCaseString(conversationId);
+    }
+
+    public String messageId() {
+        return getUUIDAsUpperCaseString(messageId);
+    }
```

Comment:
Are these really required? I would say that if all the function does is return a single UUID in uppercase - could we not just use:

```
conversationId.toUpperCase()
```

```
messageId.toUpperCase();
```

Reply:
Yea, previously we used toString().toUpperCase() here, but we extracted that to a static function as suggested by a PR comment in PRMT-3413, so probably I am keeping it like this for now.